client/landing/gutenboarding/components/header/index.tsx

```diff
@@ -9,6 +9,7 @@ import { Icon } from '@wordpress/components';
 import { useDispatch, useSelect } from '@wordpress/data';
 import { useHistory } from 'react-router-dom';
 import classnames from 'classnames';
+import { DomainSuggestions } from '@automattic/data-stores';
```

Comment:
Elsewhere in Gutenboarding it looks like we're using the following pattern to import the `DomainSuggestion` type:

```
type DomainSuggestion = import('@automattic/data-stores').DomainSuggestions.DomainSuggestion;
```

I'm not 100% sure of the benefit, but superficially it seems like that's a way to use the import only in TypeScript, without it otherwise having any effect in JS? You might know about this than I do, but since I saw the pattern used elsewhere, just thought I'd mention it!

Reply:
Good catch, I'm not sure either but that would make sense that it wouldn't get included in the compiled JS if we could import it that way, nice 👍